### DISCUSSION TITLE
A "TV Optimised" setting/mode/fork

### DISCUSSION THREAD

**Discussion by SecularSteve** [waydroid/waydroid#554] [Ideas]
I didn't think much of this before, but then I came across this:

https://plasma-bigscreen.org/
https://github.com/KDE/plasma-bigscreen

I was thinking that there could be a "Bigscreen Optimised" edition of this container that comes programmed with the following features:

-prioritise running Android apps in TV mode (if available)
-use a TV optimised build of Lineage (if available)
-libCEC optimisations
-non-ARM device optimisations
etc...

But that's just a suggestion. I just want to know if there's any potential here. No need to force it.

**Comment by jsonmona** [comment]
It would be neat. Just simply an option to run apps as TV mode would be useful.

**Comment by illegalsolutions** [comment]
Obviously such a feature would be really useful.
